examples/submit-guppy-h1-h2-via-qir.ipynb

```diff
@@ -65,12 +65,12 @@
     "\n",
     "from hugr_qir.hugr_to_qir import hugr_to_qir\n",
     "\n",
-    "guppy_qir_bitcode_string = hugr_to_qir(guppy.compile(main))\n"
```

Comment:
Is it possible to add a summary at the top saying which guppy programs can be converted to QIR and which cannot be converted?